external/vcm/vcm/calc/thermo.py

```diff
@@ -420,20 +426,24 @@ def column_integrated_heat(
 
 
 def column_integrated_heating(
-    dtemperature_dt: xr.DataArray, delp: xr.DataArray, vertical_dim: str = "z"
```

Comment:
`isochoric=False` does not imply that it is isobaric. Instead of using the flag I suggest two functions

```
column_integrated_heating_from_isobaric_transition(temperature_change_isobaric, ...)
column_integrated_heating_from_isochoric_transition(temperature_change_isochoric, ...)
```